website/docs/docs/supported-data-platforms.md

```diff
@@ -30,8 +30,8 @@ There are two types of adapters available today:
 1. Does it work?
 2. Does anyone "own" the code, or is anyone liable for ensuring it works?
 3. Do bugs get fixed quickly?
-4. Does it stay up-to-date with new Core features?
+4. Does it stay up-to-date with new dbt Core features?
 5. Is the usage substantial enough to self-sustain?
-pendency on this library?
+6. Do other well-known well-adopted projects already depend on this library?
```

Comment:
simplifying a bit bc there were a lot of 'well' and i think the intent is just 'popular' or known' projects. pls let me know if this is't the right tweak and happy to follow up @dataders !
```suggestion
6. Do other known projects depend on this library?
```